appveyor.yml

```diff
@@ -3,18 +3,15 @@
 # BSD-style license that can be found in the LICENSE file.
 
 install:
-  # Investigate scoop?
-  - choco install -y dart-sdk -version 1.11.1
+#  - choco install -y dart-sdk -version 1.11.1
+  - ps: wget https://storage.googleapis.com/dart-archive/channels/dev/release/1.12.0-dev.5.1/sdk/dartsdk-windows-x64-release.zip -OutFile dart-sdk.zip
```

Comment:
would it make sense to download whatever the latest dev channel is?

https://gsdview.appspot.com/dart-archive/channels/be/raw/latest/sdk/dartsdk-windows-x64-release.zip